authors/apps/tests/test_articles.py

```diff
@@ -54,6 +54,7 @@ def test_get_specific_article(self):
         self.assertEqual(response.status_code, 200)
         self.assertIn('articles', response.json())
         self.assertIsInstance(response.json().get("articles"), dict)
+   
```

Comment:
Remove this extra empty line

Reply:
@katunold  i have worked on the feedback